src/plugins/intel_cpu/src/emitters/snippets/aarch64/jit_binary_call_emitter.hpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <xbyak_aarch64/xbyak_aarch64/xbyak_aarch64.h>
+
+#include <cpu/aarch64/cpu_isa_traits.hpp>
+#include <cpu/aarch64/jit_generator.hpp>
+#include <cstddef>
+#include <set>
+#include <vector>
+
+#include "emitters/plugin/aarch64/jit_emitter.hpp"
+#include "snippets/emitter.hpp"
+
+namespace ov::intel_cpu::aarch64 {
+
+/**
+ * @brief Base class for binary call emitters on ARM64. Provides proper ABI compliance for function calls
+ * by managing register spilling, stack alignment, and calling conventions according to ARM64 AAPCS.
+ * This follows the same pattern as the x64 jit_binary_call_emitter but adapted for ARM64 architecture.
+ */
+class jit_binary_call_emitter : public jit_emitter {
+public:
+    jit_binary_call_emitter(dnnl::impl::cpu::aarch64::jit_generator* h,
+                            dnnl::impl::cpu::aarch64::cpu_isa_t isa,
+                            std::set<snippets::Reg> live_regs);
+
+    // Need at least one register to store the callable address
+    static size_t aux_gprs_count() {
+        return 1;
+    }
```

Comment:
Could you double check please. Looks like we don't use this method. BTW the base class `jit_emitter` has the similar virtual method `get_aux_gprs_count()`. But I think we anyway don't need this method and we can remove it